Knowledge/QA glossary.md

```diff
@@ -91,6 +110,9 @@ The glossary is meant to help you get familiar with words and phrases commonly u
 - **Hotfix** - is a critical bug fix that needs to go live before the next scheduled release date.
 - **Hover** - (or hover state) communicates when a user has placed a cursor above an interactive element. 
 - **HT (Half time)** - four working hours per day.
+- **HTTP vs HTTPS** - HTTP is a protocol or order and syntax for presenting information and is used for transferring data over a network.
+If the website you are connected to is using HTTP, your requests and responses are not encrypted and someone willing to (and knows how) can see your data.
+HTTPS is using technology for encrypting (making plaintext series of random characters) the data and authenticating both parties (verifying that the person or machine is who they claim to be). 
```

Comment:
Can you maybe put this as two separate bullets? Also, can you please write down what HTTP and HTTPS stand for?